testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/crossdc/ActionTokenCrossDCTest.java

```diff
@@ -96,7 +96,7 @@ public void sendResetPasswordEmailSuccessWorksInCrossDc(
         // KEYCLOAK-17584: Temporarily disable the test for 'community' profile till KEYCLOAK-17628 isn't fixed. In other words till:
         // * The test is either rewritten to start using the new Wildfly subsystem for base metrics introduced in Wildfly 22,
         // * Or Keycloak is able to load the Eclipse MicroProfile Metrics subsystem from the microprofile Galleon feature-pack
-        Assume.assumeTrue("Ignoring test as product profile is not enabled", Profile.getName().equals("product"));
+        Assume.assumeTrue("Ignoring test as product profile is not enabled", Profile.getInstance().getName().name().equalsIgnoreCase("product"));
```

Comment:
getName is not used frequently so doesn't have the static (convinience) method anymore.